WebApplication/ClientApp/src/components/forgePdfViewExtension.js

```diff
@@ -0,0 +1,102 @@
+// *******************************************
+// My Color Extension
```

Comment:
Leftover? :)
Please add the default header

Reply:
I will update it